libs/components/packages/src/schematics/migrations/update-9/ag-grid-29/ag-grid-29.schematic.spec.ts

```diff
@@ -0,0 +1,122 @@
+import { Tree } from '@angular-devkit/schematics';
+import { SchematicTestRunner } from '@angular-devkit/schematics/testing';
+
+describe('ag-grid-29.schematic', () => {
+  let tree: Tree;
+  const runner = new SchematicTestRunner(
+    'schematics',
+    require.resolve('../../migration-collection.json')
+  );
+
+  function setupTest(
+    packageJson: { [key: string]: any } = {},
+    build: { [key: string]: any } = {}
+  ) {
+    tree = Tree.empty();
+    const angularJson = {
+      version: 1,
+      projects: {
+        test: {
+          projectType: 'application',
+          root: '',
+          architect: {
+            build,
+          },
+        },
+      },
+    };
+    tree.create('/angular.json', JSON.stringify(angularJson));
+    tree.create('/package.json', JSON.stringify(packageJson));
+  }
+
+  it('should work', async () => {
+    expect.assertions(1);
+    setupTest(
+      {
+        dependencies: {
+          'ag-grid-community': '0.0.0',
+          'ag-grid-angular': '0.0.0',
+          'ag-grid-enterprise': '0.0.0',
+        },
+      },
+      {
+        options: {
+          styles: ['ag-grid-community/dist/styles/ag-theme-alpine.css'],
+        },
+      }
+    );
+    const updatedTree = await runner.runSchematic('ag-grid-29', {}, tree);
+    const workspace = JSON.parse(updatedTree.readText('/angular.json'));
+    expect(workspace.projects.test.architect.build.options.styles).toEqual([
+      'node_modules/@skyux/ag-grid/css/sky-ag-grid.css',
+    ]);
+  });
+
+  it('should noop', async () => {
```

Comment:
Same here. What is the schematic _not_ doing?

Reply:
Updated.